src/anyvlm/utils/types.py

```diff
@@ -0,0 +1,74 @@
+"""Provide helpful type definitions, references, and type-based operations."""
+
+from enum import Enum, StrEnum
+from typing import Annotated
+
+from pydantic import BeforeValidator, StringConstraints
+
+
+class EndpointTag(str, Enum):
+    """Define tag names for endpoints."""
+
+    META = "Meta"
+    SEARCH = "Search"
+
+
+class GrcAssemblyId(StrEnum):
+    """Supported GRC assembly identifiers"""
+
+    GRCH37 = "GRCh37"
+    GRCH38 = "GRCh38"
+
+
+class UscsAssemblyBuild(StrEnum):
+    """Supported USCS assembly builds"""
+
+    HG38 = "hg38"
+    HG19 = "hg19"
+
+
+GenomicSequence = Annotated[
+    str,
+    BeforeValidator(str.upper),
+    StringConstraints(pattern=r"^[AGCT]*$"),
```

Comment:
This is something we ran into in VRS-Python: https://github.com/ga4gh/vrs-python/issues/275 . Would be another possible place to move this to one place
```suggestion
    StringConstraints(pattern=r"^[ACGTURYKMSWBDHVN]*$"),
```

Reply:
Oh wait is there already something built in to vrs-python that I should be using? Or are you saying that there isn't yet, but there should be?